Scala/Hangman.scala

```diff
@@ -0,0 +1,294 @@
+import java.sql.{Connection, DriverManager}
+
+/**
+  * Created by Administrator on 26/06/2017.
+  */
+object Main
+{
+  def main(args: Array[String]): Unit = {

```

Comment:
This is way too long, methods should not exceed 50 lines, where this is like 200. Split the functionality into separate methods please